src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpQuickInfo.cs

```diff
@@ -29,9 +29,8 @@ static void Main(string$$[] args)
     }
 }");
             InvokeQuickInfo();
-            Assert.Equal(
-                "class\u200e System\u200e.String\r\nRepresents text as a sequence of UTF-16 code units.To browse the .NET Framework source code for this type, see the Reference Source.",
-                Editor.GetQuickInfo());
+            VerifyQuickInfo(
```

Comment:
Right. These one-line helper methods (like `VerifyQuickInfo` aren't making the tests any shorter, and they increase the complexity of the type...